lint-staged.config.js

```diff
@@ -4,6 +4,10 @@ module.exports = {
       // Exclude quantic and create-atomic-template packages
       if (file.includes('/packages/quantic/')) return false;
       if (file.includes('/packages/create-atomic-template/')) return false;
+      if (file.includes('/packages/create-atomic-component/template/'))
```

Comment:
It was not possible to commit changes that only affected files in these subpackages; biome would complain that it had processed nothing.